### DISCUSSION TITLE
CAT Control?

### DISCUSSION THREAD

**Discussion by LeSuedois** [egzumer/uv-k5-firmware-custom#483] [Q&A]
Hello,
I am wondering if it's possible to "cat control" the radio? I saw [this video](https://youtu.be/g5-S1lQo3xg), [link to github](https://github.com/phdlee/uvk5cec?tab=readme-ov-file).
And I was wondering if this is a possible with this firmware? I really love my UV-K5 with it's AIOC Cable, Raspberry Pi and WSJT-X as cheap and distant reporting rig. But since there is no CAT Control I can't program a schedule.
So there are two ways, adding the UV-K5 as radio to WSJT-X (through [Serial communication](https://github.com/nicsure/QuanshengDock) IMHO) or make it behave like a well known radio.
Anyone has a better idea?

Is there a documentation on how the UART feature works in this firmware?
If anybody can give me a hint to the documentation please?

Thank you, Simon

**Comment by ErikS-web** [comment]
As you can see here #485 is Q-Dock a Fork of EGZUMER.

And with [Q-Dock](https://github.com/nicsure/QuanshengDock/wiki), **_CAT control is now possible._**

What are your further wishes?

Tip.
Take a 2nd radio and operate it via CAT with Q-Dock (because NICSURE had to modify it as specific adapted EGZUMER firmware)
And use your first radio portable
